Project-1/TimPrudhomme_Project1.md

```diff
@@ -0,0 +1,483 @@
+```
+
+(b) `(A→(B∨C))∨(C→¬A)`
+```
+(¬A ∨ (B ∨ C)) ∨ (¬C ∨ ¬A)     [→ to ∨]
+```
+| A | B | C | (¬A ∨ (B ∨ C)) | (¬C ∨ ¬A) | (¬A ∨ (B ∨ C)) ∨ (¬C ∨ ¬A) |
+| - | - | - | -------------- | --------- | --------------------------- |
+| T | T | T | T              | F         | T                           |
+| T | F | T | T              | F         | T                           |
+| F | T | T | T              | T         | T                           |
+| F | F | T | T              | T         | T                           |
+| T | T | F | T              | T         | T                           |
+| T | F | F | F              | T         | T                           |
+| F | T | F | T              | T         | T                           |
+| F | F | F | T              | T         | T                           |
+
+This is a tautology, so its equivalent to the CNF formula `(A ∨ ¬A) ∧ (B ∨ ¬B) ∧ (C ∨ ¬C)`
+
+
+(c) `(¬A ∧ ¬B ∧ C) ∨ (¬A ∧ ¬C) ∨ (B ∧ C) ∨ A`
+```
+(¬A ∧ ¬B ∧ C) ∨ (¬A ∧ ¬C) ∨ ((B ∨ A) ∧ (A ∨ C))      [Distribute ∨]
+```
+| A | B | C | (¬A ∧ ¬B ∧ C)  | (¬A ∧ ¬C) | ((B ∨ A) ∧ (A ∨ C)) | (¬A ∧ ¬B ∧ C) ∨ (¬A ∧ ¬C) ∨ ((B ∨ A) ∧ (A ∨ C)) |
+| - | - | - | -------------- | --------- | -------------------- | ------------------------------------------------ |
+| T | T | T | F              | F         | T                    | T                                                |
+| T | F | T | F              | F         | T                    | T                                                |
+| F | T | T | F              | F         | T                    | T                                                |
+| F | F | T | T              | F         | T                    | T                                                |
+| T | T | F | F              | F         | T                    | T                                                |
+| T | F | F | F              | F         | T                    | T                                                |
+| F | T | F | F              | T         | F                    | T                                                |
+| F | F | F | F              | T         | F                    | T                                                |
+
+This is a tautology, so its equivalent to the CNF formula `(A ∨ ¬A) ∧ (B ∨ ¬B) ∧ (C ∨ ¬C)`
+
+
+---
+# Question 3
+3. Let V be the vocabulary of first-order logic consisting of a binary relation P and a unary relation F. Interpret P(x,y) as “x is a parent of y” and F(x) as “x is female.” Where possible define the following formulas in this vocabulary; where not possible, explain why: 
+```
+  (a)  B(x,y) that says that x is a brother of y
+  (b)  A(x,y) that says that x is an aunt of y
+  (c)  C(x,y) that says that x and y are cousins 
+  (d)  O(x) that says that x is an only child
+  (e)  T(x) that says that x has exactly two brothers 
+```
+
+## Answers:
+```
+FOL 
+
+// x is brother of y
+B(x,y) = ∃x∃y∃p( P(p,x) ∧ P(p,y) ∧ ¬F(x) ∧ x≠y ∧ x≠p ∧ y≠p )
```

Comment:
It's unclear whether the universally bound x,y in B(x,y) are bound by the existentials or not since the same variables were used. Typically, definitions will have universal scope by default. 

Same remarks apply to the following